src/canary.py

```diff
@@ -66,9 +66,7 @@ async def watch_events(*args, **kwargs):
     tasks = dict()
 
     try:
-        # Created monitors for current objects
-        # TODO Query kubes for existing CanaryHTTPMonitor objects that are visible and iterate over them
-        # use the context manager to close http sessions automatically
+        # Create monitors for current objects
         async with ApiClient() as api:
             crds = client.CustomObjectsApi(api)
             rawmonitors = await crds.list_cluster_custom_object(group="canary.ukserp.ac.uk", version="v1", plural="canaryhttpmonitors")
```

Comment:
So to confirm this line is working? And is returning the objects?

Reply:
Yeah, that works fine